durable-function/blog.md

```diff
@@ -2,11 +2,21 @@
 
 Azure Durable Functions is a new programming model based on Microsoft serverless' platform Azure Functions. It allows you to write a workflow as code and have the execution run with the scalability and the reliability of serverless with high throughput.
 
-As a .NET Core developer, Azure Functions v2 gives you the power to run .NET Core code with all the benefits that C# can bring to your code.
+## Scenario
+
+Initially, I wanted to index data from GitHub repositories. I explained it all in a [previous post](https://blogs.msdn.microsoft.com/appserviceteam/2018/08/06/learn-how-to-orchestrate-serverless-functions-by-scraping-apis-in-8-minutes/?WT.mc_id=dotnet-blog-marouill) but I'll drilldown into the concepts today.
+
+To summarize the post, I wanted to retrieve a list of repositories that I wanted to index. Then, query each individual repositories for more detailed information. Finally, save the data into a database.
+
+Today, we'll do something similar but to keep things simple, I'll save the content to Azure Blob Storage.
+
+If you want to follow along, I've created [a sample][samplelink] that contains all the code you will see today.
+
+Let's get started.
 
 ## Concepts
 
-Beside the normal [serverless concepts](https://docs.microsoft.com/dotnet/standard/serverless-architecture/?WT.mc_id=dotnet-blog-marouill) that you ought to know, Azure Durable Functions introduces more concepts that makes developing complex workflow easy.
+Beside the normal [serverless concepts](https://docs.microsoft.com/dotnet/standard/serverless-architecture/?WT.mc_id=dotnet-blog-marouill), Azure Durable Functions introduces more concepts that makes developing complex workflow easy.
 
 If you need to know what .NET Core is all about, we have [great tutorials](https://docs.microsoft.com/dotnet/core/tutorials/?WT.mc_id=dotnet-blog-marouill) to get you started.
```

Comment:
Plugging .NET core here feels out of place since you just told me about "Durable Functions" you haven't talked about .NET Core yet so I'd cut this sentence